src/core/src/bootstrap/EnvLayer.py

```diff
@@ -218,17 +181,13 @@ def __str__(self):
             output = subprocess.check_output(no_output, cmd, stderr=subprocess.STDOUT, shell=True)
         except subprocess.CalledProcessError as e:
             if chk_err:
-                print("Error: CalledProcessError.  Error Code is: " + str(e.returncode), file=sys.stdout)
-                print("Error: CalledProcessError.  Command string was: " + e.cmd, file=sys.stdout)
-                print("Error: CalledProcessError.  Command result was: " + self.__convert_process_output_to_ascii(e.output[:-1]), file=sys.stdout)
+                print("Error: CalledProcessError. [Code={0}][Command={1}][Result={2}]".format(str(e.returncode), e.cmd, self.__convert_process_output_to_ascii(e.output[:-1])), file=sys.stdout)
```

Comment:
Existing CC miss - difficult to unit test. Leaving out of scope while reducing verbosity.